billing.go

```diff
@@ -0,0 +1,165 @@
+package clerk
+
+// Feature represents a feature associated with a plan.
+type Feature struct {
+	APIResource
+
+	Object      string `json:"object"`
+	ID          string `json:"id"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Slug        string `json:"slug"`
+	AvatarURL   string `json:"avatar_url"`
+}
+
+// BillingMoney represents money amounts with formatting.
+type BillingMoney struct {
+	APIResource
+
+	Amount          int64  `json:"amount"`
+	AmountFormatted string `json:"amount_formatted"`
+	Currency        string `json:"currency"`
+	CurrencySymbol  string `json:"currency_symbol"`
+}
+
+// BillingProduct represents a product.
+type BillingProduct struct {
+	APIResource
+
+	Object    string  `json:"object"`
+	ID        string  `json:"id"`
+	Slug      string  `json:"slug"`
+	Currency  string  `json:"currency"`
+	Name      string  `json:"name"`
+	IsDefault bool    `json:"is_default"`
+	Plans     []*Plan `json:"plans"`
```

Comment:
```suggestion
	Plans     []Plan `json:"plans"`
```

I don't think we'll ever have a nil plan in this list, right?